lib/semantic_puppet.rb

```diff
@@ -1,7 +1,18 @@
-require 'gettext-setup'
+begin
+  require 'gettext-setup'
+  GETTEXT_AVAILABLE = true
```

Comment:
Hmm interesting. Perhaps avoid the constant altogether, instead just checking if gettext is defined? i.e., `if defined?(GettextSetup)` ? Also not foolproof